addons/pos_loyalty/models/pos_config.py

```diff
@@ -50,7 +50,7 @@ def _check_before_creating_new_session(self):
         if invalid_reward_products_msg:
             prefix_error_msg = _("To continue, make the following reward products available in Point of Sale.")
             raise UserError(f"{prefix_error_msg}\n{invalid_reward_products_msg}")
-        if  gift_card_programs:
```

Comment:
useless diff?

Reply:
Kind of, I saw two space instead of one and it triggered me